test/functional/test_framework/mininode.py

```diff
@@ -606,3 +606,19 @@ def send_txs_and_test(self, txs, node, *, success=True, expect_disconnect=False,
                 # Check that none of the txs are now in the mempool
                 for tx in txs:
                     assert tx.hash not in raw_mempool, "{} tx found in mempool".format(tx.hash)
+
+class P2PTxInvStore(P2PInterface):
```

Comment:
I prefer all classes in the framework to have a docstring, even if it's very simple. This one would be something like:

"A P2PInterface which stores a count of how many times each txid has been announced"

Reply:
done

Reply:
also added to the docstring at the top of the file.